_DATA_TYPES/_Date_Time/main/main.js

```diff
@@ -0,0 +1,293 @@
+/*
+
+Date is a built-in object.
+
+It stores the date, time and provides methods for date/time
+
+For instance, we can use it to store creation/modification times, to measure time, or just to print out
+the current date
+*/
+
+
+/*
+
+Creation
+
+To create a new Date object call new Date() with one of the following arguments:
+
+*/
+
+
+
+new Date()
+// Without arguments - create a Date object for the current date and time
+
+let now = new Date();
+
+alert( now ); // shows current date/ time
+
+new Date(milliseconds)
+// Create a Date object with the time equal to number of milliseconds (1/1000 of a second) passed after Jan 1st of 1970 UTC+0
```

Comment:
Try to limit line length to 120 characters for readability.

Reply:
Noted!